http/swagger.yml

```diff
@@ -9057,8 +9057,8 @@ components:
         orgID:
           description: the ID of the organization that owns this notification rule.
           type: string
-        authorizationID:
-          description: The ID of the authorization used to create this notification rule.
+        ownerID:
```

Comment:
@kelwang i think this should be a readonly field.  It should be sent via the POST/PATCH as owner is implied by the creator.

OwnerID would be gathered from the authorizer context

(context.Authorizer(ctx).GetUserID())